pkgs/applications/networking/instant-messengers/cinny-desktop/darwin.nix

```diff
@@ -0,0 +1,29 @@
+{ pname
+, version
+, src
+, meta
+, stdenvNoCC
+, undmg
+}:
+
+stdenvNoCC.mkDerivation {
+  inherit pname version src meta;
+
+  nativeBuildInputs = [
+    undmg
+  ];
+
+  unpackPhase = "undmg $src";
```

Comment:
There is a hook for it: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/archivers/undmg/setup-hook.sh.
```suggestion
```

Reply:
Done 👍